src/main/java/in/casekartin/model/RegisterManager.java

```diff
@@ -0,0 +1,81 @@
+package in.casekartin.model;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+public class RegisterManager {
```

Comment:
add constructor to this class.

Reply:
there i s no need to write constructor because i wrote the methods as non static